tools/editor/editor_data.cpp

```diff
@@ -611,11 +611,16 @@ void EditorData::set_edited_scene(int p_idx){
 	current_edited_scene=p_idx;
 	//swap
 }
-Node* EditorData::get_edited_scene_root(){
+Node* EditorData::get_edited_scene_root(int idx){
 
 	ERR_FAIL_INDEX_V(current_edited_scene,edited_scene.size(),NULL);
```

Comment:
Yeah I mean checking for the validity of `current_edited_scene` is only relevant when you're going to use it, so the code should be:
```
	if (idx < 0) {
		ERR_FAIL_INDEX_V(current_edited_scene,edited_scene.size(),NULL);
		return edited_scene[current_edited_scene].root;
	} else {
		ERR_FAIL_INDEX_V(idx,edited_scene.size(),NULL);
		return edited_scene[idx].root;
	}